src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -124,18 +142,38 @@ private void SnapToNewSnapshot(IIssuesSnapshot newSnapshot)
 
         public void RequestAnalysis(IAnalyzerOptions options)
         {
-            FilePath = document.FilePath; // Refresh the stored file path in case the document has been renamed
+            RequestAnalysisAsync(options).Forget();
```

Comment:
Same pattern as for other cases where we want to switch from a `void` method to an `async` one.